### PR TITLE
Add `__version__` to Python module.

### PR DESCRIPTION
**Pull request by kevinzakka**
Add the ability to print the current version of MuJoCo.

```python
>>> import mujoco
>>> mujoco.__version__
'2.1.3'
```

### CODE REVIEW

File: python/mujoco/__init__.py [hunk 1]
```diff
@@ -79,3 +79,14 @@
...
+
+def _get_version() -> str:
+  with open(os.path.join(HEADERS_DIR, 'mujoco.h'), 'r') as f:
+    for line in f:
+      if line.startswith('#define mjVERSION_HEADER'):
+        version = line.split()[2]
+        break
+  return '.'.join([d for d in str(version)])
+
+__version__ = _get_version()
```
[the remark this answers is not in the document]
Comment (kevinzakka) [reply python/mujoco/__init__.py:92]: Oops, I only discovered this last night while pouring over the docs :)

File: python/mujoco/__init__.py [hunk 2]
```diff
@@ -79,3 +79,8 @@
       from mujoco.egl import GLContext
     else:
       from mujoco.glfw import GLContext
+
+
+from mujoco import mj_versionString
```
Comment (saran-t) [inline python/mujoco/__init__.py:84]: You don't need this line: the `from mujoco._functions import *` line should've taken care of it already.

**Review by saran-t — CHANGES_REQUESTED** [review]

File: python/mujoco/__init__.py (hunk 2 above)
Comment (kevinzakka) [reply python/mujoco/__init__.py]: Done.

File: python/mujoco/__init__.py [hunk 3]
```diff
@@ -79,3 +79,6 @@
       from mujoco.egl import GLContext
     else:
       from mujoco.glfw import GLContext
+
+
```
Comment (saran-t) [inline python/mujoco/__init__.py:83]: nit: remove one extraneous empty line

**Review by saran-t — CHANGES_REQUESTED** [review]

**Review by saran-t — APPROVED** [review]

**Comment by saran-t** [comment]
It might take me a little while to get this merged on our side. We can't use GitHub merge since our source of truth is Google's internal monorepo, and right now it looks like our Copybara workflow isn't correctly configured to accept PR (hence the error you see above).

**Pull request merged**